File: Beginner/Mahasena (AMR15A)/mahasena.java
import java.util.*;


class Coddechef {
	public static void main (String[] args) throws java.lang.Exception {
        
		Scanner scan=new Scanner(System.in);
		if(scan.hasNext()){
		    int t = scan.nextInt();
		    int even = 0;
		    int odd = 0;
		    for(int i=0;i<t;i++){
		        int num = scan.nextInt();
		        if(num%2==0)
		            even++;
		        else
		            odd++;
		    }
		    if(even>odd){
		        System.out.println("READY FOR BATTLE");
		    }
		    else{
		        System.out.println("NOT READY");
		    }
        }
        scan.close();
	}
}

File: Beginner/Mahasena (AMR15A)/mahasena.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int n,a[100];
    cin >> n;
    int odd=0, even=0;
    for(int i=0;i<n;i++) {    
      cin>>a[i];
        if(a[i]%2==0)
        even++;
        else
        odd++;
    }
    if(even>odd) {
        cout<<"READY FOR BATTLE";
    }
    else {
        cout<<" NOT READY";
    }
    return 0;
}